packages/at_demo_data/pubspec.yaml

```diff
@@ -4,10 +4,10 @@ homepage: https://www.atsign.dev
 documentation: https://atsign.dev/docs/
 repository: https://github.com/atsign-foundation/at_demos
 
-version: 1.0.4
+version: 1.1.0
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=3.0.0 <4.0.0'
```

Comment:
If I do a `dart pub get` on this package the `pubspec.lock` gets an sdk constraint like this (after tree shaking):

```
sdks:
  dart: ">=3.4.0 <4.0.0"
```

Something that deserves an arch conversation is how realistic our notional support for earlier Dart versions is.